tests/e2escenarios/e2e_source_test.go

```diff
@@ -36,47 +36,47 @@ var _ = Describe("odo source e2e tests", func() {
 		// issue https://github.com/openshift/odo/issues/4623
 		// It("Should be able to deploy a wildfly source application", func() {
 		// 	helper.CopyExample(filepath.Join("source", "wildfly"), commonVar.Context)
-		// 	helper.CmdShouldPass("odo", "create", "--s2i", "wildfly", "wildfly-app", "--project",
-		// 		commonVar.Project, "--context", commonVar.Context)
+		// 	helper.Cmd("odo", "create", "--s2i", "wildfly", "wildfly-app", "--project",
+		// 		commonVar.Project, "--context", commonVar.Context).ShouldPass()
 
 		// 	// Push changes
-		// 	helper.CmdShouldPass("odo", "push", "--context", commonVar.Context)
-		// 	cmpList := helper.CmdShouldPass("odo", "list", "--context", commonVar.Context)
-		// 	Expect(cmpList).To(ContainSubstring("wildfly-app"))
+		// 	helper.Cmd("odo", "push", "--context", commonVar.Context)
+		// 	cmpList := helper.Cmd("odo", "list", "--context", commonVar.Context).ShouldPass()
```

Comment:
Missing .out as you are need to assign the output to cmpList

Reply:
those are commented test so i left them, I will update them in my next PR where I remove the unused helper functions

Reply:
We run the risk to miss this required change if we leave it pending for your next PR. My suggestion is to update the functionality on disabled tests as well, so we can get done with this required changes now.
If you prefer to change them at a later time, then add a comment that this will need to be done.

Reply:
okay understood I will add a comment on the issue that commented test are not changed.

Reply:
I would say if we are changin half of the part, why not add `.out` as well.

Reply:
updated.